src/template.rs

```diff
@@ -112,12 +112,115 @@ pub mod django_rusty_templates {
     }
 
     impl Engine {
+        fn get_template_loaders<'py>(
+            py: Python<'py>,
+            template_loaders: &Bound<'_, PyList>,
+        ) -> Result<Vec<Loader>, PyErr> {
+            template_loaders
+                .iter()
+                .map(|template_loader| Self::find_template_loader(py, template_loader))
+                .collect()
+        }
+
         fn find_template_loader<'py>(
-            _py: Python<'py>,
-            _loader: &str,
-            _args: Option<Bound<'py, PyAny>>,
-        ) -> PyResult<Bound<'py, PyAny>> {
-            todo!()
+            py: Python<'py>,
+            ld: Bound<'_, PyAny>,
+        ) -> Result<Loader, PyErr> {
+            // Try as string first
+            if let Ok(loader_str) = ld.downcast::<PyString>() {
+                return Self::map_loader(
+                    py,
+                    &loader_str.extract::<String>()?,
+                    Vec::new(),
+                    HashMap::new(),
+                );
+            }
+
+            // Try as tuple (loader, args)
+            if let Ok(loader_tuple) = ld.downcast::<PyTuple>() {
+                if let Ok((loader_obj, args)) =
+                    loader_tuple.extract::<(Bound<'_, PyString>, Bound<'_, PyAny>)>()
```

Comment:
The [Python code we're re-implementing](https://github.com/django/django/blob/42ab99309d347f617d60751c2e8d627fb2963049/django/template/engine.py#L142-L143) does:

```python
        if isinstance(loader, (tuple, list)):
            loader, *args = loader
```

but the Rust code here only handles the case where `loader` has two elements. I think we should write a helper function:

```rust
fn unpack(loader: Bound<'py, PyAny>) -> PyResult<(Bound<'py, PyString>, Vec<Bound<'py, PyAny>>)> {
    let mut items = loader.try_iter()?;
    let loader_path = items.next().downcast()?;
    Ok((loader_path, items.collect()))
}
```